picard/util/tags.py

```diff
@@ -844,14 +1463,22 @@ def script_variable_tag_names():
     yield from (
         tagvar.script_name()
         for tagvar in ALL_TAGS
-        if tagvar.is_script_variable
+        if not tagvar.not_script_variable
```

Comment:
This change can be reverted since we keep `is_script_variable`.

Reply:
Good catch!  I completely missed that.